### PR TITLE
PHP 8.1 fix

### PR DESCRIPTION
**Pull request by 3Dgoo**
This addresses issue https://github.com/silverstripe/silverstripe-multiuser-editing-alert/issues/21

### CODE REVIEW

File: src/MultiUserEditingController.php [hunk 1]
```diff
@@ -75,7 +69,7 @@ public function set($request)
 
         $dataArray = array();
         $dataArray['lastEdited'] = date("Y-m-d H:i:s"); //when this was last updated
-        $dataArray['abbreviatedName'] = $this->user->FirstName . ' ' . substr($this->user->Surname, 0, 1);
+        $dataArray['abbreviatedName'] = $this->user->FirstName . ' ' . ($this->user->Surname ? substr($this->user->Surname, 0, 1) : '');
```
Comment (michalkleiner) [inline src/MultiUserEditingController.php]:
```suggestion
        $dataArray['abbreviatedName'] = $this->user->FirstName . ' ' . substr($this->user->Surname ?: '', 0, 1);
```

File: src/MultiUserEditingController.php [hunk 2]
```diff
@@ -33,13 +33,7 @@ protected function init()
...
         $this->editingCache = Injector::inst()->get(CacheInterface::class . '.multiuserediting');
 
-
-        $usersEditing = unserialize($this->editingCache->get('editing'));
-
-        //create a new simple PHP object to store user editing data
-        if (!$usersEditing) {
-            $usersEditing = array();
-        }
+        $usersEditing = $this->editingCache->get('editing') ? unserialize($this->editingCache->get('editing')) : array();
```
Comment (michalkleiner) [inline src/MultiUserEditingController.php]:
```suggestion
        $usersEditing = $this->editingCache->get('editing') ? unserialize($this->editingCache->get('editing')) : [];
```

File: src/MultiUserEditingController.php (hunk 1 above)
Comment (3Dgoo) [reply src/MultiUserEditingController.php]: Of course. That is much cleaner. Thanks. I have made this change.

File: src/MultiUserEditingController.php (hunk 2 above)
Comment (3Dgoo) [reply src/MultiUserEditingController.php]: Yes. Sorry. I just copy and pasted the existing array() code. I have made this change now.

**Review by michalkleiner — APPROVED** [review]

**Pull request merged**